player/command.c

```diff
@@ -5855,7 +5855,7 @@ static void cmd_mouse(void *p)
     button += dbc ? MP_MBTN_DBL_BASE : MP_MBTN_BASE;
     if (pre_key)
         mp_input_put_key_artificial(mpctx->input, pre_key);
-    mp_input_set_mouse_pos_artificial(mpctx->input, x, y);
+    mp_input_set_mouse_pos_artificial(mpctx->input, x, y, 0);
```

Comment:
Is there any value to allowing this command to pass modifiers? I'm honestly not sure, but I'd thought I'd just mention it.

Reply:
I think it would make sense, yes. It occurred to me as well, but then I realized that this command does not allow you to pass modifiers to artificial button clicks either yet, so maybe as a separate enhancement?

Reply:
If the change is just as simple as adding a new argument to this input command, than it should be part of the same commit imo.